packages/bot-web-ui/src/pages/bot-builder/quick-strategy/selects/__tests__/symbol.spec.tsx

```diff
@@ -106,18 +101,51 @@ describe('<SymbolSelect />', () => {
         const option_element = screen.getByText(/Bear Market Index/i);
         userEvent.click(option_element);
 
-        expect(autocomplete_element).toHaveDisplayValue([/Bear Market Index/i]);
+        expect(autocomplete_element).toHaveValue('Bear Market Index');
+    });
+
+    it('should select item from the list if item is not found', async () => {
+        mock_store.ui.is_desktop = true;
+        render(<SymbolSelect />, {
+            wrapper,
+        });
+
+        const autocomplete_element = screen.getByTestId('dt_qs_symbol');
+        userEvent.click(autocomplete_element);
+
+        expect(autocomplete_element).not.toHaveValue('Bear Market Index');
+    });
+
+    it('should handle hide dropdown list', async () => {
+        mock_store.ui.is_desktop = true;
+        render(<SymbolSelect />, { wrapper });
+
+        const autocomplete_element = screen.getByTestId('dt_qs_symbol');
+        userEvent.click(autocomplete_element);
+        userEvent.type(autocomplete_element, 'Invalid');
```

Comment:
@farabi-deriv do we need to click and type or just start typing would do both?

Reply:
@shafin-deriv just user event type is fine. Removed click event.